packages/shared/prisma/schema.prisma

```diff
@@ -422,6 +426,34 @@ enum ScoreSource {
     EVAL
 }
 
+model ScoreConfig {
+    id          String              @id @default(cuid())
+    createdAt   DateTime            @default(now()) @map("created_at")
+    updatedAt   DateTime            @default(now()) @updatedAt @map("updated_at")
+    projectId   String              @map("project_id")
+    project     Project             @relation(fields: [projectId], references: [id], onDelete: Cascade)
+    name        String
+    dataType    ScoreConfigDataType @map("data_type")
+    isArchived  Boolean             @default(false) @map("is_archived")
+    minValue    Float?              @map("min_value")
+    maxValue    Float?              @map("max_value")
+    categories  Json?               @map("categories")
+    description String?
+    score       Score[]
+
+    @@unique([id, projectId]) // used for upserts via prisma
+    @@index([dataType])
+    @@index([isArchived])
+    @@index([projectId])
+    @@index([categories])
+    @@map("score_configs")
+}
+
+enum ScoreConfigDataType {
```

Comment:
simplify to ScoreDataType? This is tough to edit once there are tables that rely on this

Reply:
I'd prefer `ScoreConfigDataType` as we have our `scores` table which I would expect to hold this data type when called `ScoreDataType`, not the score configs table. Another suggestion could be `ConfigDataType`, but that would be too generic imo. What would be the reason for simplifying? I think we'd loose context in doing so.

Reply:
No strong opinion here. I'd see this as the config constraints a score to a datatype

Reply:
OK, would you be happy to leave as `ScoreConfigDataType` then? Then we can get this merged ☺️